### PR TITLE
feat(patterns/dividers): add dividers

### PR DESCRIPTION
**Pull request by nherwegh**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [X] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [X] No

## Describe the changes

Add the divider pattern

GitHub issue number or Jira issue URL: [ID-1904](https://adeo.kanbanize.com/ctrl_board/10/cards/1904/details/)

### CODE REVIEW

**Comment by tiloyi** [comment]
Deployment link : https://featdividers-dot-demo-dot-mozaic-design-system-dtrp.ey.r.appspot.com

File: src/docs/Components/Dividers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  This version will only be visible on high end screens and won't be seen by people who have sight troubles.
+
+</Highlight>
+
+### Default
+
+This is the version you should use in your layouts. This default version is AAA accessible and will be visible by every users.
+
+### Dark
```
Comment (tiloyi) [inline src/docs/Components/Dividers/index.mdx:36]: Could the variations be categorized from those that should be used most often to those that should be used least often? Maybe the "default" first and then the others (Dark, Light, Lightest)?

Comment (caqueste) [reply src/docs/Components/Dividers/index.mdx:36]: I agree with you @tiloyi

Comment (nherwegh) [reply src/docs/Components/Dividers/index.mdx:36]: @tiloyi yes, good idea 👍

**Pull request merged**